htlcswitch/link.go

```diff
@@ -516,7 +516,14 @@ func (l *channelLink) Stop() {
 		l.cfg.ChainEvents.Cancel()
 	}
 
-	l.updateFeeTimer.Stop()
+	// Ensure the channel for the timer is drained.
+	if !l.updateFeeTimer.Stop() {
```

Comment:
Nice! Did this actually happen?

Reply:
Actually I wasn't sure...was playing with `pprof` and saw this code so I made the change😂